rest_framework_jwt/views.py

```diff
@@ -3,6 +3,8 @@
 from rest_framework.response import Response
 from datetime import datetime
 
+from django.middleware import csrf
```

Comment:
just a consideration as I saw this referred to at https://github.com/Styria-Digital/django-rest-framework-jwt/issues/4 , maybe it is appropriate to only import this if `api_settings.CSRF_COOKIE` is enabled, e.g. as a delayed import within the `if` below.  

Then any site not using csrf middleware doesnt get hit with an extra module being import that isnt used.  Are there many sites doing that?

Also pep8 tools would complain bitterly about that.